src/editor/codemirror/linting/linter.worker.ts

```diff
@@ -0,0 +1,19 @@
+import * as TPyParser from "tigerpython-parser";
+
+TPyParser.setLanguage("en");
+TPyParser.rejectDeadCode = true;
+TPyParser.strictCode = true;
+
+/* eslint-disable no-restricted-globals */
+let workerSelf = self as unknown as BroadcastChannel;
```

Comment:
I don't think this is really the right interface but it had the correct definition of postMessage. Is there something saner I can do short of splitting this out into a separate project?

Reply:
2813f39940c9e9dae986ae41e4787fd2b5f5ec09